mlir/lib/ExecutionEngine/conv-validation-wrappers.cpp

```diff
@@ -930,6 +931,9 @@ void mcpuVerify(T *gpuResults, T *validationResults, int64_t dataSize,
       }
       // Accumulate square root
       sumDiffSq += static_cast<double>(absDiff) * static_cast<double>(absDiff);
+      // Print out values if difference is larger than threshold
+      if (absDiff > thr_absDiff || relDiff > thr_relDiff)
+        printf("%ld: %f %f %f %lf\n", i, valNum, gpuNum, absDiff, relDiff);
```

Comment:
@krzysz00 It is printed here.

Reply:
Oh, lol, I'm absolutely brilliant at this. And so that's basically all my comments, I think this is a good improvement!